template/android/app/build.gradle

```diff
@@ -190,20 +190,25 @@ android {
             // If you wish to add a custom TurboModule or component locally,
             // you should uncomment this line.
             // preBuild.dependsOn("generateCodegenArtifactsFromSchema")
-            preDebugBuild.dependsOn(packageReactNdkDebugLibs)
-            preReleaseBuild.dependsOn(packageReactNdkReleaseLibs)
+            android.applicationVariants.all { def variant ->
```

Comment:
Why is this needed? The code was originally written under the assumption that you had both debug & release build type (which is safe to assume, as those are the default build types).

Are you removing one of the two?

Reply:
I'm not, but I've pushed a commit fixing a bug with the arch specific ones. Basic idea here is that a flavorless app typically has `preDebugBuild` & `preReleaseBuild` whereas an app with flavors has `pre$FlavorDebugBuild` & `pre$FlavorReleaseBuild`.